packages/sdk/contractkit/package.json

```diff
@@ -53,5 +53,8 @@
   },
   "engines": {
     "node": ">=8.13.0"
-  }
+  },
+  "browser":{
+    "child_process": false
+  }  
```

Comment:
Is this change intended?

Reply:
yes! I forgot where exactly, but there's a line in contractkit that doesn't work in the app if this isn't included